### PR TITLE
chore: Fixed auto-release script

### LINKED ISSUE #2797: Fix the auto release script.
**Describe the chore**
Fix the auto release script.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by yashk2000**
Fixed #2797 

Changes: Attempt to fix the auto release script.

### CODE REVIEW

**Comment by yashk2000** [comment]
@iamareebjamal please review.

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -29,8 +29,8 @@ fi
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     echo "Push to master branch detected, signing the app..."
     cp app-release-unsigned.apk app-release-unaligned.apk
-    jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
-    ${ANDROID_HOME}/build-tools/26.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
+    jarsigner -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
+	${ANDROID_HOME}/build-tools/26.0.3/zipalign -p 4 app-release-unaligned.apk app-release.apk
```
Comment (liveHarshit) [inline scripts/update-apk.sh]: I think 26.0.3 should replace with `$BUILD_TOOLS_VERSION` to solve the `directory not found` Travis error.

Comment (yashk2000) [reply scripts/update-apk.sh]: @liveHarshit ok. But I do not understand how it makes a difference.

Comment (liveHarshit) [reply scripts/update-apk.sh]: Build successful, build tool version is the same :+1:

Comment (liveHarshit) [reply scripts/update-apk.sh]: I think the build tool version is different, in Travis script it is  `ANDROID_BUILD_TOOLS_VERSION=25.0.3`

Comment (liveHarshit) [reply scripts/update-apk.sh]: Just test it by running the full script, and exit before the release commands.

Comment (yashk2000) [reply scripts/update-apk.sh]: @liveHarshit sorry for the delay in response. Thanks for the help. I'll try updating the `ANDROID_BUILD_TOOLS_VERSION` to `28.0.3` and then running the script.

**Comment by yashk2000** [comment]
@liveHarshit @iamareebjamal is this fine?

**Comment by yashk2000** [comment]
@iamareebjamal this commit got merged with master by mistake. Can we revert it?

**Comment by iamareebjamal** [comment]
It was deliberate

**Pull request merged**